functions.php

```diff
@@ -154,6 +154,11 @@ function c4aa_setup_theme_supported_features() {
 			'slug' => 'caa-grey',
 			'color' => '#968c83',
 		),
+		array(
+			'name' => __( 'C4AA Black', 'themeLangDomain' ),
```

Comment:
Update all instances of `themeLangDomain` with `c4aa`

Reply:
Huh? Watcha mean exactly?

Reply:
@slambert Any place that says `'themeLangDomain' should be 'c4aa' - that 'themeLangDomain' is a placeholder from the tutorial!

Reply:
Ah thank you!